contracts/interfaces/i_emission_manager.cairo

```diff
@@ -0,0 +1,45 @@
+%lang starknet
+
+from contracts.types.rewards_data import RewardsDataTypes
+
+@contract_interface
+namespace IEmissionManager:
+    func configure_assets(config_len : felt, config : RewardsDataTypes.RewardsConfigInput*):
```

Comment:
probably better to rename `config` to `configs` or `inputs`

Reply:
We could, I'm just following the same name as in original contracts, what do you think?

Reply:
oh ok, very good then! I was surprised to see singular instead of plural, but let's keep it like that if it's singular on solidity side